app/views/stripe/_js.html.erb

```diff
@@ -1,8 +1,8 @@
 <% if Rails.application.config.stripe.debug_js %>
-<script type="text/javascript" src="https://js.stripe.com/v1/stripe-debug.js"></script>
+<script type="text/javascript" src="https://js.stripe.com/v2/stripe-debug.js"></script>
 <% else %>
-<script type="text/javascript" src="https://js.stripe.com/v1/"></script>
+<script type="text/javascript" src="https://js.stripe.com/v2/"></script>
 <% end %>
 <script type="text/javascript">
-Stripe.setPublishableKey("<%= Rails.application.config.stripe.publishable_key or fail 'No stripe.com publishable key found. Please set config.stripe.publishable_key in config/application.rb to one of your publishable keys, which can be found here: https://manage.stripe.com/#account/apikeys' %>")
-</script>
\ No newline at end of file
+Stripe.setPublishableKey("<%= Rails.application.config.stripe.publishable_key || raise 'No stripe.com publishable key found. Please set config.stripe.publishable_key in config/application.rb to one of your publishable keys, which can be found here: https://manage.stripe.com/#account/apikeys' %>")
```

Comment:
I had to fix this to `raise('No stripe.com ...')` otherwise I get a syntax error.